templates/otp.conf.j2

```diff
@@ -0,0 +1,9 @@
+description "start OpenTripPlanner process"
+
+start on {{ upstart_start_on }}
+stop on runlevel [!2345]
+
+chdir {{ otp_bin_dir }}
+script
+    exec sudo -u {{ otp_user }} ./otp -a --graphs {{ otp_data_dir }} --server
```

Comment:
`setuid` in the Upstart script should handle dropping to the `otp_user` instead of using `sudo -u`.